docs/index.md

```diff
@@ -100,23 +100,21 @@ SpacetimeDB was designed first and foremost as the backend for multiplayer Unity
 1. How do I get/install SpacetimeDB?
    Just install our command line tool and then upload your application to the cloud.
 
-1. How do I create a new database with SpacetimeDB?
+4. How do I create a new database with SpacetimeDB?
```

Comment:
why do we switch to explicit numbering here?

Reply:
> why do we switch to explicit numbering here?

Editor shenanigans, likely - fixing >>

Reply:
> why do we switch to explicit numbering here?

![image](https://github.com/clockworklabs/spacetime-docs/assets/8840024/171fe075-237f-4609-a4a9-c33ff1e1af78)
Ah, I see now - there was a lone `5.` below a bash script that wouldn't unhinged from the implicit.

I'll 1~5 them explicitly.